pkg/cloudprovider/aws/amifamily/bootstrap/bootstrap.go

```diff
@@ -29,6 +31,7 @@ type Options struct {
 	Labels                  map[string]string `hash:"set"`
 	CABundle                *string
 	AWSENILimitedPodDensity bool
+	KubeDnsIP               net.IP
```

Comment:
Is it possible to just use the value in KubeletConfig?

Reply:
Using the value in the kubelet config would depend on the user passing that optional parameter. So if we relied on this, we'd need to default it based on our discovery or require the user to pass in cluster dns ip to engage the ipv6 functionality. Requiring the user to specify the dns ip would be very surprising. 

I do agree that adding another `kubeDNSIP` to the bootstrap struct is probably not the right path here though since you'd need to decide which dns ip to use in a bootstrap implementation.

Reply:
I appreciate your feedback. I agree that the additional valuables in the struct is not appropriate. I moved the DNS discovery to `eksbootstrap` because the PR is only related to `bootstrap.sh` of AMI.